### PR TITLE
Refine the IconSet Block

### LINKED ISSUE #91: Refine IconSet Module

### PR DESCRIPTION
**Pull request by ethanzitting**
Extract the quantity and content of the icons to state to facility a future CMS.

Strengthen the IconSet and Icon blocks for potential widespread use.

Closes #91

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -1,13 +1,41 @@
...
 
 export default function About(): JSX.Element {
+    const teamIcons: Icon[] = [
+        {
+            image: {
+                src: '/images/man-profile-2.png',
+                alt: 'Meetup Logo',
+            },
+            title: <div className={'h3 ' + styles.name}>Fred Lawler</div>,
+            subtitle: <div className={'h6 ' + styles.position}>Co-Organizer</div>,
```
Comment (fredlawl) [inline src/pages/about.tsx:14]: I'm curious if this is more semantic: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

File: src/pages/about.tsx [hunk 2]
```diff
@@ -1,13 +1,41 @@
...
+
     return (
         <div className={styles.pageWrapper}>
             <h1>About Us</h1>
-            <IconSet />
-            <p>
+            <IconSet icons={teamIcons} />
+            <p className={styles.groupDescription}>
                 A group who takes on tech volunteer work around the community. We meet every Tuesday and host occasional
                 socials. Connect with us and{' '}
```
Comment (fredlawl) [inline src/pages/about.tsx:40]: `and {' '}` ? Is this intentional? If so, `&nbsp;` is the encoding for space.

File: src/pages/volunteer.tsx [hunk 3]
```diff
@@ -30,11 +31,32 @@ export default function volunteer(): JSX.Element {
...
+            },
+            title: <div className="h1">Meetup</div>,
+        },
+        {
+            image: {
+                url: 'https://discord.gg/jFD8dZP',
+                src: '/images/discord-logo.svg',
+                alt: 'Discord Logo',
+            },
+            title: <div className="h1">Discord</div>,
```
Comment (fredlawl) [inline src/pages/volunteer.tsx:49]: Similar to profile avatars, maybe prefer `<figure><figcaption>` with inner  `<span className="h1">`  here.

File: src/pages/about.tsx (hunk 1 above)
Comment (fredlawl) [reply src/pages/about.tsx:14]: In general, if you need block level elements, perfer `<p>` or something else. Divs should be reserved for ad-hoc structure where other block elements don't make sense.

Comment (ethanzitting) [reply src/pages/about.tsx:14]: These are both really helpful! I will be implementing them.

**Pull request merged**